modnet/featurizers/presets/debreuck_2020.py

```diff
@@ -113,12 +113,15 @@ class DeBreuck2020Featurizer(modnet.featurizers.MODFeaturizer):
         VoronoiFingerprint(),
     )
 
+    def __init__(self,fast_oxid=False):
+        self.fast_oxid = fast_oxid
+    
```

Comment:
Also need to add this to the composition-only preset

Reply:
It seems composition-only preset doesn't use any oxid featurizer.

Reply:
This must be an oversight right? We should probably add it as a general method of the featurizer anyway so that all oxidation features can be controlled in this way

Reply:
I will raise another issue for this